src/v2/spec.yaml

```diff
@@ -72,6 +72,17 @@ components:
         The query param accepted by the API. Sent along with identifier param.
         url - Cast identifier is a url
         hash - Cast identifier is a hash
+    FilterSpam:
```

Comment:
we dont expose spam filtering to developers yet, we only have it internally right now. lets take this out and we will expose it once we are ready